api/src/main/java/net/kyori/adventure/bossbar/BossBar.java

```diff
@@ -31,7 +31,15 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
- * A bossbar.
+ * Represents an in-game bossbar which can be shown to the client.
+ * <p>A boss bar consists of:</p>
+ * <ul>
```

Comment:
You should use a `<dl>` definition list for these lists -- it better captures the split between field name and definition.